thirdparty/imgui_suite/BUILD.bazel

```diff
@@ -0,0 +1,356 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("@rules_python//python:defs.bzl", "py_binary")
+load("//shared/bazel/rules:objectivec_rules.bzl", "wpilib_objc_library")
+
+cc_library(
+    name = "gl3w",
+    srcs = ["generated/gl3w/src/gl3w.c"],
+    hdrs = [
+        "generated/gl3w/include/GL/gl3w.h",
+        "generated/gl3w/include/GL/glcorearb.h",
+        "generated/gl3w/include/KHR/khrplatform.h",
+    ],
+    includes = [
+        "generated/gl3w/include",
+    ],
+    strip_include_prefix = "generated/gl3w/include",
+)
+
+cc_library(
+    name = "fonts",
+    srcs = glob(["generated/fonts/src/**"]),
+    hdrs = glob(["generated/fonts/include/**"]),
+    includes = ["generated/fonts/include"],
+    deps = [
+        ":imgui",
+    ],
+)
+
+py_binary(
+    name = "gl3w_gen",
+    srcs = ["gl3w/gl3w_gen.py"],
+    tags = ["manual"],
```

Comment:
Any reason this needs to be manual?  Building it is cheap, running it needs a bazel run.

Reply:
If it is not manual, when you build with a cross compiler it fails because it can't find a good version of python. This can probably be fixed with the `target_compatible_with`